hermes/bin/src/runtime_extensions/hermes/integration_test/event.rs

```diff
@@ -70,11 +70,15 @@ impl HermesEventPayload for OnBenchEvent {
 /// Fails to execute an event.
 #[allow(dead_code)]
 pub fn execute_event(
-    module: &mut Module, test: u32, run: bool,
+    module: &mut Module, test: u32, run: bool, bench: bool,
```

Comment:
use an enum here instead of `bench: bool`.  Its a choice between two things, not a flag enabling bench or not, so its clearer if its an enum.